src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,195 @@
+  </Step>
+
+  <Step>
+    ## Set up alerting
+
+    You can manage your <InlinePopover type="alerts" />, [create](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/) new ones, configure your [notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications/), identify [anomalies](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/), configure [correlation policies](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions), and more. 
+
+    Some of the [quickstarts](https://newrelic.com/instant-observability) include pre-configured alerts that save you time setting up monitoring tools.
+
+    You can also check [our tutorial](/docs/tutorial-create-alerts/create-new-relic-alerts/) on getting started with New Relic alerts.
+    
+  </Step>
+
+  <Step>
+
+    ## Add users
+
+    You can add the rest of your team to New Relic. Use the **User management** UI to [add users](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where). See [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) to learn more about the different types of users. You can also check [our tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial) on adding users and managing user access.
+  </Step>
+</Steps>
+
+
+
+# Capabilities [#capabities]
+
+Our platform offers a comprehensive suite of capabilities to help organizations monitor, troubleshoot, and optimize the performance of their digital environments. Here's an explanation of New Relic's key capabilities. If you want to learn more, explore our [tutorials](/docs/new-relic-solutions/tutorial-landing-page/) pages. 
+
+
+## Alerts & AI [#alerts]
+
+Alerts help you identify issues in your applications and code. They show you how these issues affect your users. When you receive notifications of your alerts, you can stop problems and prevent widespread user disruptions. Being aware of issues early can lead to better user experiences.
+
+Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue.
+
+<img
+  title="New Relic alerts main page"
+  alt="New Relic alerts main page"
+  src={NRAlertsMain}
+/>
+
+
+
+## APM & Services [#apm]
+
+New Relic instruments your application at the code level through the use of one of our many language agents. These agents collect metrics from your application and send them to our application performance monitoring (APM) , allowing you to monitor it using pre-built dashboards.
```

Comment:
What's important about APM are all the languages supported as well as how it can be deployed on a wide range of systems, including Kubernetes and Docker.

I think it would be good to list all of the languages here